src/openms/source/MATH/STATISTICS/PosteriorErrorProbabilityModel.cpp

```diff
@@ -961,6 +965,7 @@ namespace OpenMS
       const StringList search_engines = ListUtils::create<String>("XTandem,OMSSA,MASCOT,SpectraST,MyriMatch,"
                                                                   "SimTandem,MSGFPlus,MS-GF+,Comet,"
                                                                   "tide-search,SimpleSearchEngine,"
+                                                                  "MSFragger"
```

Comment:
I think this introduced a bug.
should be `"MSFragger,"` ... and since this way of specifying a list of strings is obviously not very robust, even better would be C++17 brace init:
`const StringList search_engines = { "XTandem", "OMSSA", ... }` :)

Reply:
Yes @tillenglert please fix.

Reply:
Sorry for that inconvenience, I haven't checked thorough enough and missed this. My new PR should fix it.
About that list: I tried to keep everything as it is and just include my changes, but I can include this in my next PR, thanks for that :)